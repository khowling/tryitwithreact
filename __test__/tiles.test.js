// Copyright 2004-present Facebook. All Rights Reserved.
/* eslint-disable no-unused-vars */

'use strict';

import React from 'react';
import App from '../app/app.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const component =  renderer.create(
    <App buildprops={{ server_url: 'http://localhost:3000'}}/>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot();
});

