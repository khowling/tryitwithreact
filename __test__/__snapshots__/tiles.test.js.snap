exports[`test renders correctly 1`] = `
<div
  className="slds">
  <section
    className="site-banner">
    <div
      className="slds-container--center slds-container--medium" />
  </section>
  <div
    style={
      Object {
        "height": "3.5rem",
      }
    } />
  <div
    className="container">
    not booted
    <div
      className="slds-spinner--small">
      <img
        alt="Loading..."
        src="http://localhost:3000/assets/images/spinners/slds_spinner.gif" />
    </div>
  </div>
</div>
`;
